cypress/integration/analytics.test.js

```diff
@@ -64,6 +64,52 @@ describe('add_to_cart event', () => {
   })
 })
 
+describe('remove_from_cart event', () => {
```

Comment:
Could you add a test to check if the sent values (such as product_reference_id and sku_name) are right?

Reply:
Hmm my answer would be yes, _but_:
- I'm thinking about how could I retrieve those values from the UI (to compare them with what is in the dataLayer). I had to add the `'data-sku'` attribute (at `useRemoveButton.ts`) and I fear that adding more just for the sake of testing will bloat the return of this hook. Thoughts/other solutions?
- Even if those values were available in the UI for comparison, isn't it a bit weird to check for other attributes? (considering that my idea was to make this test only the minimum required things to make it work):
  - Those attributes are optional, and also the dev can at any time choose which attributes to send or not, if they erase some attribute the test would break pointlessly
  - I'm already checking the `sku` attribute not for checking the payload "shape", just for checking that the removed item is the same item in the data layer
  - I'm already checking the required attributes `currency` and `item_name`, and if other dev removes those, the test breaks because the payload is no longer valid.

Reply:
About how to retrieve the data from UI, you can use `cy.intercept`.  I did a test using it on `analytics.test.js`

https://docs.cypress.io/api/commands/intercept

Reply:
To me, it makes sense to test other attributes too (even if they are not mandatory) and check if they match the removed product. But this is just a suggestion, I also understand your point, so feel free to ignore it 😄